```
Traceback (most recent call last):
  File "manage.py", line 88, in <module>
    execute_from_command_line(sys.argv)
  File "/vendor/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/vendor/lib/python2.7/site-packages/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/vendor/lib/python2.7/site-packages/django/core/management/commands/test.py", line 50, in run_from_argv
    super(Command, self).run_from_argv(argv)
  File "/vendor/lib/python2.7/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/vendor/lib/python2.7/site-packages/django/core/management/commands/test.py", line 71, in execute
    super(Command, self).execute(*args, **options)
  File "/vendor/lib/python2.7/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/vendor/lib/python2.7/site-packages/django/core/management/commands/test.py", line 88, in handle
    failures = test_runner.run_tests(test_labels)
  File "/mnt/testrunner.py", line 385, in run_tests
    all_suite = self.build_suite(all_test_labels, extra_tests)
  File "/vendor/lib/python2.7/site-packages/django/test/simple.py", line 242, in build_suite
    suite.addTest(build_suite(app_config))
  File "/vendor/lib/python2.7/site-packages/django/test/simple.py", line 144, in build_suite
    tests_module = get_tests(app_config)
  File "/vendor/lib/python2.7/site-packages/django/test/simple.py", line 102, in get_tests
    test_module = import_module('%s.%s' % (app_config.name, TEST_MODULE))
  File "/usr/local/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/mnt/testapps/test_pillowtop/tests/__init__.py", line 3, in <module>
    from .test_reindexer import *
  File "/mnt/testapps/test_pillowtop/tests/test_reindexer.py", line 6, in <module>
    from corehq.apps.domain.tests.test_utils import delete_all_domains
ImportError: No module named test_utils
```

--- a/corehq/apps/domain/tests/test_utils.py
+++ b/corehq/apps/domain/tests/test_utils.py
@@ -1,0 +1,8 @@
+from corehq.apps.domain.models import Domain
+from corehq.util.test_utils import unit_testing_only
+
+
+@unit_testing_only
+def delete_all_domains():
+    domains = Domain.get_all()
+    Domain.bulk_delete(domains)

